content/basic-use.md

```diff
@@ -0,0 +1,182 @@
+# Basic use
+
+In their day to day work, most Ember developers use only a small number of CLI commands.
+We'll cover them here, along with a quick tutorial of how to use the `--help` option.
+
+## Using the help command
+
+For any of the commands below, you can see all of the options available by using the `--help` flag.
+
+For example, `ember generate --help` will show you a full list of all the types of files you can create using the CLI.
+
+<!-- SCREENSHOT -->
+
+## Create an app
+
+### Format:
+
+```
+ember new <my-app-name> [options]
+```
+
+### What it does
+
+`ember new` creates all the boilerplate files that are part of an Ember app. It puts them in a folder with the same name as whatever we provide in place of `<my-app-name>`.
+
+### Example use
+
+This command below will create a folder called `camping-trip-tracker`, which will be full of app files. It uses the `--yarn` option to show that the app should use `yarn` by default. Yarn is a package manager alternative to `npm`. Options typically start with a double dash `--` and can be omitted entirely.
+
+```
+ember new camping-trip-tracker --yarn
+```
+
+### Learn more
+
+<!-- Link to quickstart in the guides -->
+
+## Serve the app locally
+
+### Format
+
+```
+ember serve [options]
+```
+
+You can also use shorthand like `ember s`.
```

Comment:
It might be helpful to mention shorthands at the top of this section, since they can be used for any command. Then maybe just simply list specific shorthands under each command. For example, I also use `ember g` in place of `generate`.